pom.xml

```diff
@@ -75,7 +75,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<integration-test.pattern>**/*IntegrationTest*</integration-test.pattern>
 		<!-- exclude preview module code from sonar cloud analysis -->
-		<sonar.exclusions>spring-cloud-previews/!(google-cloud-language-spring-starter)/**</sonar.exclusions>
```

Comment:
Maybe you needed to escape the `-` with `\-`?

Reply:
Oh possibly...my other suspicion is that maybe `/**` only matches directories, and `/**/*` is still needed to match files?

Retrying this change in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1853, but will merge this revert for now. Thanks!